### ISSUE TITLE
Vulnerability in graphql-paper > immer dependency

### ISSUE THREAD

**Issue by wfischer42** [graphql-mocks/graphql-mocks#104]
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ critical      │ Prototype Pollution in immer                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ immer                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=9.0.6                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ graphql-paper                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ graphql-paper > immer                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1002492                     │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

Resolving immer to `^9.0.6` doesn't seem to cause any problems, so I think it'll work as a drop-in upgrade. Haven't run the tests though.

**Comment by chadian** [comment]
Looks like everything was passing in #105. It'll get out in the next release.

**Comment by chadian** [comment]
```
❯ npm info graphql-paper

graphql-paper@0.1.3 | MIT | deps: 5 | versions: 5

dist
.tarball: https://registry.npmjs.org/graphql-paper/-/graphql-paper-0.1.3.tgz
.shasum: 81629a386412049ccd652b36680c05cb25c1a200
.integrity: sha512-7YIMQJZ5/pOaD1pa6RVWvnnqpgFt3WKzSwG+SKdU3AFVUsynXVqEEwQwHNPzonwgxn5fNF69kybUBLb15rrgPA==
.unpackedSize: 229.2 kB

dependencies:
event-target-polyfill: ^0.0.3 immer: ^9.0.6                 just-diff: ^3.1.1             lodash.merge: ^4.6.2          short-unique-id: ^4.3.3
```

This released in #117.
